lib/flipper/action_patch.rb

```diff
@@ -12,9 +12,9 @@ def view(name)
 
       raise "Template does not exist: #{path}" unless path.exist?
 
-      contents = path.read
-      compiled = Flipper::UI::Eruby.new(contents)
-      compiled.result proc {}.binding
+      # rubocop:disable Lint/Eval
+      eval(Erubi::Engine.new(path.read, escape: true).src)
+      # rubocop:enable Lint/Eval
```

Comment:
[following example in Flipper gem](https://github.com/jnunemaker/flipper/pull/407/files)